flathub.json

```
@@ -0,0 +1,4 @@
+{
+	"skip-icons-check": true,
+	"only-arches": [ "x86_64", "i386" ]
```

```suggestion
	"only-arches": [ "x86_64" ]
```


we haven't supported i386 for a few years.

---

Didn't know that, I used https://github.com/flathub/com.obsproject.Studio.Plugin.InputOverlay/blob/master/flathub.json as an "inspiration" so that's why that is there. I'll remove it then.

---

no idea why it's there.